docker-image-src/5/coredb/Dockerfile-ubi8

```diff
@@ -11,7 +10,7 @@ RUN set -eux; \
             tinisha="93dcc18adc78c65a028a84799ecf8ad40c936fdfc5f2a57b1acda5a8117fa82c"; \
             ;; \
         'aarch64') \
-            tiniurl="https://github.com/krallin/tini/releases/download/v0.19.0/tini-arm64"; \
+            tiniurl="https://github.com/krallin/tini/releases/download/v0.19.0/tini-arm64"; \nh
```

Comment:
looks like some keys got mashed at the end there. That's probably gonna break the build.